gsy_framework/schema/avro_schemas/launch_simulation_scenario.json

```diff
@@ -0,0 +1,341 @@
+{
+  "name": "ScenarioArea",
+  "type": "record",
+  "namespace": "my.types",
```

Comment:
Can you please explain this?
From the documentation, I only read `namespace, a JSON string that qualifies the name;` Which I do not understand.

Reply:
This is a remnant from an older implementation and I will remove it. In general, namespaces are used in order to reference types from different schemas. That way another schema could refer to this one like this: "my.types.InfiniteBus" for instance. I will remove it though since we do not need this functionality yet, and I will add it when needed.

Reply:
Sound good! Thanks!